build.sbt

```diff
@@ -127,7 +127,7 @@ lazy val lsp4s = project
   .settings(crossScalaVersions := List(V.scala211, V.scala212))
   .dependsOn(jsonrpc)
 
```

Comment:
Can we set `name := "metalsRoot"` in the top-level? By default sbt names the root project after the directory name, unless there is a project with a conflicting in which case it creates a bogus name like "default-fd2bf4". This name appears in IntelliJ after importing the build so I think "metalsRoot" would be nice

Reply:
👍 done. Thanks for explaining where the bogus name came from, also the sbt launcher displays it and I was confused.